### PR TITLE
FEAT: Added Product-listing_filters page

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-20 22-46-50

### CODE REVIEW

File: Components/ProductListing-filters/productlist.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+.filter-section-col {
+    display: flex;
+    flex-direction: column;
+    width: 27%;
+    background-color: var(--white);
+}
+
+.price-slider {
+    padding: 10px 2rem;
```
Comment (AtulHande03a) [inline Components/ProductListing-filters/productlist.css]: use consistent units.. px or rem..this looks a little odd

File: Components/ProductListing-filters/productlist.css [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+.btn-link {
+    border: none;
+    padding: 0px;
+}
+
+.filter-section-col {
+    display: flex;
+    flex-direction: column;
+    width: 27%;
```
Comment (chanchal16) [inline Components/ProductListing-filters/productlist.css]: Here you can give width in rems.

File: Components/ProductListing-filters/productlist.css [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+.product-container {
+    display: flex;
+    flex-flow: row wrap;
+    margin-top: 2rem;
+    margin-left: 4rem;
+}
+
+.product-header {
+    margin: 2rem 0 0 6rem;
```
Comment (AtulHande03a) [inline Components/ProductListing-filters/productlist.css:158]: you could have used margin top and margin left ..

**Review by AtulHande03a — APPROVED** [review]
apart from the comments rest looks good!!

File: Components/ProductListing-filters/productlist.css (hunk 1 above)
Comment (chanchal16) [inline Components/ProductListing-filters/productlist.css]: Don't mix the units. Either use px or rem.

**Review by NeerajVodala — APPROVED** [review]
Looks good to me(I wouldn't use mixed units, please check with others once before merging)
All the best

**Review by sagargse — APPROVED** [review]
all looks good to me

File: Components/ProductListing-filters/productlist.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css]: Sure will update the same

File: Components/ProductListing-filters/productlist.css (hunk 3 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css:158]: I have preferred short-hand notation there.

File: Components/ProductListing-filters/productlist.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css]: Sure, will update it 👍🏻

File: Components/ProductListing-filters/productlist.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/ProductListing-filters/productlist.css]: Sure 👍🏻

**Comment by kotesh-arya** [comment]
Thanks for these great suggestions guys.

**Pull request merged**